common/info.ts

```diff
@@ -0,0 +1,3 @@
+import packageJson from '../package.json' with { type: "json" };
+export const MCP_SERVER_NAME = "SmartBear MCP Server";
```

Comment:
What about using `description` from the `package.json` for this, or using a [custom](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#config) `config` entry?

Reply:
yeh, updated to add config to package.json